framework/py/flwr/common/inflatable_protobuf_utils.py

```diff
@@ -99,3 +105,37 @@ def push_object_fn(object_id: str, object_content: bytes) -> None:
             raise ObjectIdNotPreregisteredError(object_id)
 
     return push_object_fn
+
+
+def make_confirm_message_received_fn_protobuf(
+    confirm_message_received_protobuf: ConfirmMessageReceivedProtobuf,
+    node: Node,
+    run_id: int,
+) -> Callable[[str], None]:
+    """Create a confirm message received function that uses gRPC.
+
+    Parameters
+    ----------
+    confirm_message_received_protobuf : ConfirmMessageReceivedProtobuf
```

Comment:
would it be better to call it `confirm_message_received_stub_method` ? I only got to understand how this would work when looking into https://github.com/adap/flower/pull/5599